tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossy_with_snappi.py

```diff
@@ -67,6 +67,11 @@ def test_pfc_pause_single_lossy_prio(snappi_api,                # noqa: F811
     snappi_extra_params = SnappiTestParams()
     snappi_extra_params.multi_dut_params.multi_dut_ports = snappi_ports
 
+    flow_factor = 1
+
+    if snappi_ports[0]['duthost'].facts()['asic_type'] == 'cisco-8800' and snappi_ports[0]['speed'] == '400000':
+        flow_factor = 2
```

Comment:
We can use 
if snappi_ports[0]['duthost'].facts()['asic_type'] == 'cisco-8800':
    if int(snappi_ports[0]['speed']) > 200000:
        flow_factor = int(snappi_ports[0]['speed']) / 200000.0

To account for possible future 800G speeds. @sdszhang, @abdosi : pls comment.

Reply:
@rraghav-cisco  i have updated the logic